pkgs/top-level/python-packages.nix

```diff
@@ -1730,6 +1730,8 @@ in {
     };
   };
 
+  comp = callPackage ../applications/video/comp { };
```

Comment:
This is an application and should therefore be called directly from `all-packages.nix`, and not via `python-packages.nix`.